src/components/squareComponent/square.spec.js

```diff
@@ -1,17 +1,90 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import GameSquare from './squareComponent';
-import Enzyme from 'enzyme';
+import Enzyme, { mount } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 
 Enzyme.configure({ adapter: new Adapter() });
 
 describe('Square Component', () => {
-
   it('renders the Square component without crashing', () => {
```

Comment:
This test was almost certainly useful to get started, however is probably no longer needed. I would delete this test.

Reply:
This test has been removed